### DISCUSSION TITLE
Ability to filter NFTs

### DISCUSSION THREAD

**Discussion by jhxyz** [devdotxyz/eth-xyz#47] [Ideas]
Edit: Updating this to from "Some users do not want to display all the NFTs from OpenSea #47
" to "Ability to filter NFTs" - because a filtering feature will help with this.

A user brought up that they can be airdropped spam to their addresses, and that they would not want this spam to be displayed.

They can manually hide these somehow already. I am not sure what the solution is, given that airdropping spam is not something eth.xyz can control, nor I am guessing can OpenSea. 

Any ideas or feedback welcome!

**Comment by jhxyz** [comment]
https://twitter.com/tjthorne_photo/status/1473022253323325442?s=20

https://twitter.com/kairong__/status/1473115847153819648?s=20

https://twitter.com/thefaketomato/status/1473317714634625030?s=21

**Comment by brendandebeasi** [comment]
I don't see any indication of an asset being hidden, nor a way to filter by hidden/not hidden when making a request of OS's API. This would be something OpenSea would need to implement or fix on their end.

I have reached out to OpenSea regarding this update.

**Reply by jhxyz** [reply]
Thanks! I've seen support for both sides - some people appreciate the transparent nature of showing all NFTs. Others would like to hide their mistakes and/or spam sent to their addresses.

Let's hear more from the community on this before making any changes.

**Comment by jhxyz** [comment]
Some people have requested that the NFTs be organized like Rainbow.me
